xml/System.Diagnostics.Tracing/EventCounter.xml

```diff
@@ -109,8 +109,14 @@ For an example of how to use the <xref:System.Diagnostics.Tracing.EventCounter>
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Removes the counter from the set that the <see cref="T:System.Diagnostics.Tracing.EventSource" /> will report on.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+## Remarks
+
+After being disposed, this counter will stop reporting values. If an EventCounter is not explicitly disposed it will stop reporting automatically when the EventSource it is attached to is disposed.
```

Comment:
```suggestion
After being disposed, this counter will stop reporting values. If an event counter is not explicitly disposed it will stop reporting automatically when the event source it is attached to is disposed.
```